samples/tab-channel-group-config-page-auth/csharp/ConfigTabAuthentication/Views/Home/Index.cshtml

```diff
@@ -0,0 +1 @@
+﻿<h4>Welcome..</h4>
```

Comment:
Please remove if we are not using this file.

Reply:
Done